models/settings_helpers.go

```diff
@@ -109,6 +113,9 @@ func UpdateSettings(q reform.DBTX, params *ChangeSettingsParams) (*Settings, err
 		settings.AlertManagerURL = ""
 	}
 
+	// because of validation, both variables cannot be true at the same time
+	settings.EnableSTT = params.EnableSTT && !params.DisableSTT
```

Comment:
settings via API can be passed independently, So, both values can be `false`. Please update like we did for telemetry

Reply:
We also need validate using params and saved settings.
if `settings.Telemetry.Disabled` is true `params.EnableSTT` can't be true.
if `settings.STTEnabled` is true `params.DisableTelemetry` can't be true.